hostprocess/flannel/kube-proxy/kube-proxy.yml

```diff
@@ -21,14 +21,16 @@ spec:
           runAsUserName: "NT AUTHORITY\\system"
       hostNetwork: true
       containers:
-      - image: sigwindowstools/kube-proxy:v1.22.4-flannel-hostprocess
+      - image: sigwindowstools/kube-proxy:v1.24.3-flannel-hostprocess
         name: kube-proxy
         imagePullPolicy: Always
         env:
         # kube-proxy will use this value to select the correct network
         # https://github.com/kubernetes/kubernetes/blob/b0bc8adbc2178e15872f9ef040355c51c45d04bb/pkg/proxy/winkernel/proxier.go#L155-L158
         - name: KUBE_NETWORK
           value: "flannel.4096"
+        - name: CNI_BIN_PATH
+          value: C:\\Program Files\\containerd\\cni\\bin
```

Comment:
This path doesn't match with the path that the `Install-Containerd.ps1` path:

https://github.com/kubernetes-sigs/sig-windows-tools/blob/f366740c551b4217887a9eb339fd7ec3d6caacce/kubeadm/scripts/Install-Containerd.ps1#L123-L124